docs/zh/developer-guide/entity-permission-extension.md

```
@@ -164,6 +164,45 @@ _collect_entity_authorized_kbs(user_id, accessible_groups)
     → 收集角色和分组信息
 ```
 
+### 群组 Namespace 的实体成员
+
+除了 KnowledgeBase、Team 等资源外，**Namespace（群组）本身也支持实体成员绑定**。通过将部门等外部实体绑定为群组成员，该实体下的所有用户自动获得群组权限，并继承访问群组内下游资源（知识库、Team）的权限。
+
+**核心流程：**
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)